crucible/src/Lang/Crucible/CFG/Expr.hs

```diff
@@ -765,6 +765,38 @@ data App (ext :: Type) (f :: CrucibleType -> Type) (tp :: CrucibleType) where
          -> !(f (BVType w)) -- The shift amount as an unsigned integer.
          -> App ext f (BVType w)
 
+  -- Rotate left
+  BVRol :: (1 <= w)
+        => !(NatRepr w)
+        -> !(f (BVType w)) -- Value to rotate
+        -> !(f (BVType w)) -- The rotate amount as an unsigned integer
+        -> App ext f (BVType w)
+
+  -- Rotate right
+  BVRor :: (1 <= w)
+        => !(NatRepr w)
+        -> !(f (BVType w)) -- Value to rotate
+        -> !(f (BVType w)) -- The rotate amount as an unsigned integer
+        -> App ext f (BVType w)
+
+  -- count leading zeros
+  BVClz :: (1 <= w)
+        => !(NatRepr w)
+        -> !(f (BVType w))
+        -> App ext f (BVType w)
+
+  -- count trailing zeros
+  BVCtz :: (1 <= w)
```

Comment:
Mnemonically `BVClz` initially made me think "clear with zeroes".  Since these are new, maybe something a little more suggestive like `BVCntLZ` and `BVCntTZ`

Reply:
Sure, we can use longer names here.